cpp/src/TrinoBridge.cpp

```diff
@@ -851,6 +861,9 @@ JNIEXPORT jstring JNICALL Java_io_trino_jni_TrinoBridge_getTaskStats(JNIEnv* env
           io::trino::protocol::TaskStats taskStats = getTaskStats(task, taskId);
           nlohmann::json j;
           io::trino::protocol::to_json(j, taskStats);
+          if (counter++ % 100 == 0) {
+            std::cout << "========= task stats: " << j.dump().c_str() << std::endl;
```

Comment:
Should we use `LOG(INFO)`?

Reply:
oh, just debug log. commit this by mistake, will revert.